src/test/java/htmlverification/tests/CvsPsvPrsBilingualTest.java

```diff
@@ -0,0 +1,179 @@
+        certificatePageObjectVTP30Welsh = new CertificatePageObject(certHtmlVTP30Welsh);
+    }
+
+    @Test
+    public void verifyResultSummaries() {
+        String resultName = certificatePageObjectVTP20.getDefectSummaryComponent().getResultNameItem().text();
+        String resultNameWelsh = certificatePageObjectVTP20Welsh.getDefectSummaryComponent().getResultNameItem().text();
+
+        assertEquals(
+                String.format("%s%s%s %s", "(", EU_NUMBER_SUMMARY_HEADER, ")", PASS_WITH_DEFECTS_HEADER),
+                resultName);
+        assertEquals(
+                String.format("%s%s%s %s", "(", EU_NUMBER_SUMMARY_HEADER, ")", PASS_WITH_DEFECTS_HEADER_WELSH),
+                resultNameWelsh);
+    }
+
+    @Test
+    public void verifyCountryOfRegistration() {
+        String vtp20CountryOfRegistration = certificatePageObjectVTP20.getCountryOfRegistration();
+        String vtp30CountryOfRegistration = certificatePageObjectVTP30.getCountryOfRegistration();
+        String vtp20WelshCountryOfRegistration = certificatePageObjectVTP20Welsh.getCountryOfRegistration();
+        String vtp30WelshCountryOfRegistration = certificatePageObjectVTP30Welsh.getCountryOfRegistration();
+
+        assertEquals(testCertificate.getData().getCountryOfRegistrationCode(), vtp20CountryOfRegistration);
+        assertEquals(testCertificate.getData().getCountryOfRegistrationCode(), vtp30CountryOfRegistration);
+        assertEquals(testCertificate.getFailData().getCountryOfRegistrationCode(), vtp20WelshCountryOfRegistration);
+        assertEquals(testCertificate.getFailData().getCountryOfRegistrationCode(), vtp30WelshCountryOfRegistration);
+    }
+
+    @Test
+    public void verifyMakeAndModel() {
+        String vtp20MakeAndModel = certificatePageObjectVTP20.getMakeAndModel();
+        String vtp30MakeAndModel = certificatePageObjectVTP30.getMakeAndModel();
+        String vtp20WelshMakeAndModel = certificatePageObjectVTP20Welsh.getMakeAndModel();
+        String vtp30WelshMakeAndModel = certificatePageObjectVTP30Welsh.getMakeAndModel();
+
+        assertEquals(
+                String.format("%s%s %s", testCertificate.getData().getMake(), ",", testCertificate.getData().getModel()),
+                vtp20MakeAndModel
+        );
+        assertEquals(
+                String.format("%s%s %s", testCertificate.getFailData().getMake(), ",", testCertificate.getFailData().getModel()),
+                vtp30MakeAndModel
+        );
+
+        assertEquals(
+                String.format("%s%s %s", testCertificate.getData().getMake(), ",", testCertificate.getData().getModel()),
+                vtp20WelshMakeAndModel
+        );
+        assertEquals(
+                String.format("%s%s %s", testCertificate.getFailData().getMake(), ",", testCertificate.getFailData().getModel()),
+                vtp30WelshMakeAndModel
+        );
+    }
+
+    @Test
+    public void verifyAdvisoryDefects() {
```

Comment:
I think it will be worthwhile also verifying major/majorWelsh and dangerous/dangerousWelsh are behaving as they should, given those defects are what would cause a PRS / PassFail certificate to be generated.